### PR TITLE
Reinstate node disk size in custom launch template.

### PR DESCRIPTION
**Pull request by sengi**
Missed from #1228.

https://www.github.com/terraform-aws-modules/terraform-aws-eks/blob/v19.21.0/modules/eks-managed-node-group/main.tf#L332

### CODE REVIEW

File: terraform/deployments/cluster-infrastructure/main.tf [hunk 1]
```diff
@@ -34,13 +34,25 @@ locals {
...
+      # TODO(#1201): remove disk_size and use_custom_launch_template after AL2023 rollout.
+      use_custom_launch_template = var.govuk_environment == "integration"
       disk_size                  = var.node_disk_size
-      use_custom_launch_template = var.govuk_environment == "integration" # TODO(#1201): remove with AL2023 rollout.
-      update_config              = { max_unavailable = 1 }
+      block_device_mappings = {
+        xvda = {
+          device_name = "/dev/xvda"
+          ebs = {
+            volume_size           = 60
```
Comment (dj-maisy) [inline terraform/deployments/cluster-infrastructure/main.tf:49]: Do we want to set this to `var.node_disk_size` or shall we leave this hard-coded for now?

**Review by dj-maisy — APPROVED** [review]
One optional comment otherwise all good. 👍

File: terraform/deployments/cluster-infrastructure/main.tf (hunk 1 above)
Comment (dj-maisy) [reply terraform/deployments/cluster-infrastructure/main.tf:49]: Oh - also, should we add the same mapping to the ARM group?

Comment (sengi) [reply terraform/deployments/cluster-infrastructure/main.tf]: Whoops, good catches, thanks!

**Comment by sengi** [comment]
Checked what's up with the `EbsOptimized=false` thing and it's fine; [it's a no-op for modern instance types](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-optimized.html#current).

**Pull request merged**